src/main/webapp/index.html

```diff
@@ -28,7 +28,7 @@
         <li><a href="/">Home</a></li>
       </ul>
     </nav>
-    <h1>CodeU Starter Project</h1>
+    <h1>(=^･ｪ･^=))ﾉ彡☆ CodeU Starter Project</h1>
```

Comment:
This change is unrelated to the stats page, right? Could you make this a separate pull request?

Reply:
I don't think I can; I'm sorry, everything that happened since I pull requested was updated in this pull request. :( Let me know if you find a way, and I'd be glad to.